src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private String filepath;
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Return a list of tasks saved in local storage.
+     * @return List of tasks saved in local storage.
+     * @throws DukeException If there is error in the local storage file.
+     */
+    public List<String> load() throws DukeException {
+        try {
+
+            FileReader fr = new FileReader(filepath);
+            BufferedReader br = new BufferedReader(fr);
+
+            String task;
+            List<String> tasks = new ArrayList<>();
+            while ((task = br.readLine()) != null) {
+                tasks.add(task.strip());
+            }
+            br.close();
+            fr.close();
+
+            return tasks;
+
+        } catch (IOException e) {
+            throw new DukeException("OOPS! I'm sorry, there is an error during initialisation :-(");
+        }
+    }
+
+    /**
+     * Creates new storage file for saving the tasks.
+     * @throws DukeException If there is error while creating the local file.
+     */
+    public void newStorage() throws DukeException {
```

Comment:
Should the method name `newStorage` start with a verb? I think I prefer `initializeNewStorage`.